tokenomics/elixir.md

```diff
@@ -0,0 +1,31 @@
+# Elixir
+
+Elixir is the long term liquidity solution for Dai, EtherDai and MKR. It is a liquidity pool token of one third DAI, one third liquid staked Ether tokens and one third MKR. Elixir is used to provide sustainable liquidity on ethereum mainnet and all L2 networks. 
```

Comment:
Wondering if it's worth mentioning L2 here, seems a little confusing as a reader (is the LPing happening on the L2s? How does Maker ensure liquidity across "all" L2s?) 

Might make a bit more sense to say "Elixir is intended to ensure sustainable liquidity for MKR and the protocol's primary products, DAI and ETHD"